src/scsmodule.c

```diff
@@ -33,9 +33,14 @@ struct ScsPyData {
   PyArrayObject *c;
 };
 
+
+PyObject *solve_lin_sys_cb = SCS_NULL;
+PyObject *accum_by_a_cb = SCS_NULL;
+PyObject *accum_by_atrans_cb = SCS_NULL;
+
 /* Note, Python3.x may require special handling for the scs_int and scs_float
  * types. */
-static int get_int_type(void) {
```

Comment:
Is there a problem with these being static?

Reply:
I'm using them as `extern` in my linsys solver, although I could just copy them over there or share them in a cleaner way if you prefer

Reply:
Ok that's fine, probably good to prefix them with something like 'scs_' just so they don't collide with any other possible functions in the namespace.

Reply:
Added https://github.com/bamos/scs-python/commit/67aa92eccfd53576902fcf22b892547652ebcb63